src/components/Skeleton/Skeleton.tsx

```diff
@@ -9,8 +9,25 @@ const b = block('skeleton');
 
 export interface SkeletonProps
     extends Pick<React.HTMLAttributes<HTMLDivElement>, 'className' | 'style'>,
-        QAProps {}
+        QAProps {
+    /**
+     * Controls whether the animation effect is enabled
+     * @default true
+     */
+    isAnimated?: boolean;
```

Comment:
Lets use reverted boolean flag, for example `disableAnimation`. It is more convenient when flag is falsy by default